pkg/exporter/exporter.go

```diff
@@ -659,6 +684,8 @@ func (e *Exporter) parseStats(ch chan<- prometheus.Metric, stats map[net.Addr]me
 		}
 
 		err = firstError(
+			e.parseTimevalAndNewMetric(ch, e.rusageUser, prometheus.GaugeValue, s, "rusage_user"),
+			e.parseTimevalAndNewMetric(ch, e.rusageSystem, prometheus.GaugeValue, s, "rusage_system"),
```

Comment:
Shouldn't these be CounterValue?
```suggestion
			e.parseTimevalAndNewMetric(ch, e.rusageUser, prometheus.CounterValue, s, "rusage_user"),
			e.parseTimevalAndNewMetric(ch, e.rusageSystem, prometheus.CounterValue, s, "rusage_system"),
```

Reply:
Thank you for reviewing!
These are countValue.